### DISCUSSION TITLE
How to Edit TEXT after having saved file in OpenRaster

### DISCUSSION THREAD

**Discussion by Joaquim3** [PintaProject/Pinta#1127] [Q&A]
Hello,

My config :  Pinta 2.1.2

I would like to edit a TEXT after having saved my file in OpenRaster ?
Is it possible ?

If not, is there a way to do it ?

Kind Regards,

**Comment by cameronwhite** [comment]
If you're still working in Pinta the text might have been finalized (you can see this in the History pane) - in that case you could undo and the text will be editable again

But if you're just opening a `.ora` file from disk, there's no way to edit the text directly since .ora files just contain the final image data for each layer. You'd have to remove the old text and re-create it

**Reply by Joaquim3** [reply]
@cameronwhite
Thank you very much
